mysite/ShareBox/views.py

```diff
@@ -1,7 +1,38 @@
 from django.http import HttpResponse
+from ShareBox.models import Snippet
+from django.views.decorators.csrf import csrf_exempt
 import requests
 import httplib
+import json
 
 
-def send_data(request, snippet_data):
-    pass
+@csrf_exempt
+def insert_db(request):
+    if request.method == "POST":
```

Comment:
A better way to do this would be using the require_http_method decorator. https://docs.djangoproject.com/en/1.9/topics/http/decorators/#allowed-http-methods